NDTensors/src/lib/TypeParameterAccessors/src/Base/iswrappedarray.jl

```diff
@@ -0,0 +1,71 @@
+using SimpleTraits
+using LinearAlgebra:
+  Adjoint,
+  Diagonal,
+  Hermitian,
+  LowerTriangular,
+  Symmetric,
+  Transpose,
+  UnitLowerTriangular,
+  UnitUpperTriangular,
+  UpperTriangular
+using Base: ReshapedArray, SubArray
+using StridedViews: StridedView
+# Trait indicating if the AbstractArray type is an array wrapper.
+# Assumes that it implements `NDTensors.parenttype`.
+@traitdef IsWrappedArray{ArrayT}
+
+#! format: off
+@traitimpl IsWrappedArray{ArrayT} <- is_wrapped_array(ArrayT)
+#! format: on
+
+"""
+Optional definitions for types which are considered `Wrappers` and have a `parenttype`
+
+  Should return a `Position`.
```

Comment:
Should return `Int`?

Reply:
Yes this should say int. I will read through this file again, I just wanted to push so you could review my changes. Thanks!